test/tokenizer.test.ts

```diff
@@ -1,106 +0,0 @@
-/*
-*                      Copyright 2023 Salto Labs Ltd.
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with
-* the License.  You may obtain a copy of the License at
-*
-*     http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*/
-import { extractFormulaIdentifiers } from '../src/tokenizer'
-
-describe('extractFormulaIdentifiers', () => {
-  describe('when there are no identifiers', () => {
-    it('should skip arithmetic expressions', () => {
-      expect(extractFormulaIdentifiers('2 == (1.5 + 2) || (4 == 2) + 2')).toHaveLength(0)
-    })
-    it('should skip boolean literals', () => {
-      expect(extractFormulaIdentifiers('TRUE || False || true && (false)')).toHaveLength(0)
-    })
-    it('should skip string literals', () => {
-      expect(extractFormulaIdentifiers('\'a\' + \'b\'')).toHaveLength(0)
-    })
-    it('should skip string literals even if they have escaped quotes', () => {
-      expect(extractFormulaIdentifiers('\'\\\'a\' + "\\"b"')).toHaveLength(0)
-    })
-    it('should skip function calls', () => {
-      expect(extractFormulaIdentifiers('Function()')).toHaveLength(0)
-    })
-    it('should skip comments', () => {
-      const formula = `
-      /*identifier*/
```

Comment:
Can you also add multi-line comments as in:
```
/*
a
b
c
*/
```

Reply:
https://github.com/salto-io/salesforce-formula-parser/pull/4